etc/generate-openapi.sh

```diff
@@ -4,13 +4,15 @@ declare -r ETC_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 declare -r ROOT_DIR="$(dirname ${ETC_DIR})"
 
 declare -r GENERATOR_DOCKER_IMG=openapitools/openapi-generator-cli:v5.1.0
-declare -r OPENAPI_COMMIT=e41d5e327a67e472a46cd6edfe673496e1b066dd
+declare -r OPENAPI_COMMIT=sgc/issues/9672
```

Comment:
When you reset this and re-generate, the changes from #57 should take effect. I expect the only impact will be that the new files added here will get renamed to `*.gen.go`